cpp/sqliteBridge.cpp

```diff
@@ -307,11 +487,11 @@ SequelResult sequel_execute(jsi::Runtime &rt, string const dbName, string const
         case SQLITE_INTEGER:
         {
           /**
-           * It's not possible to send a int64_t in a jsi::Value because JS cannot represent the whole number range. 
```

Comment:
Would be nice to have auto-formatting working on XCode, this applies I think only to my case, since I code on XCode to get auto completions... maybe figure out how to set up XCode properly to detect headers